package.json

```diff
@@ -27,7 +27,7 @@
   ],
   "dependencies": {
     "backbone": "1.2.3",
-    "cartodb.js": "CartoDB/cartodb.js#f0bd380",
+    "cartodb.js": "CartoDB/cartodb.js#5aa87cf",
```

Comment:
Are we still using a fixed version for a reason? I mean, couldn't we use `"CartoDB/cartodb.js#v4"`?